<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>544</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>编译原理项目</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_window" columnstretch="2,5">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="inputLabel">
        <property name="font">
         <font>
          <family>华文中宋</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>输入文本：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="inputText">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="options">
        <property name="font">
         <font>
          <family>华文中宋</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="title">
         <string>选项</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_option">
         <item row="0" column="0">
          <widget class="QPushButton" name="btnSelectFile">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>华文中宋</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>选择文件</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnSaveFile">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>华文中宋</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>另存至文件</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="Answer">
      <property name="font">
       <font>
        <family>华文中宋</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="WordAnal">
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_WordAnal">
        <item row="0" column="0">
         <widget class="QGroupBox" name="GroupWord">
          <property name="font">
           <font>
            <family>华文中宋</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="title">
           <string>功能按钮</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>30</number>
           </property>
           <property name="topMargin">
            <number>30</number>
           </property>
           <property name="rightMargin">
            <number>30</number>
           </property>
           <property name="bottomMargin">
            <number>30</number>
           </property>
           <property name="spacing">
            <number>11</number>
           </property>
           <item row="0" column="2">
            <widget class="QPushButton" name="btnDfa">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>华文中宋</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>DFA</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="btnSdfa">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>华文中宋</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>最小化DFA</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btnNfa">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>华文中宋</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>NFA</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="btnWordAnal">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>华文中宋</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>词法分析程序</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="GrammarAnal">
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_GramAnal">
        <item row="0" column="0">
         <widget class="QGroupBox" name="GroupGrammar">
          <property name="font">
           <font>
            <family>华文中宋</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="title">
           <string>功能按钮</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>30</number>
           </property>
           <property name="topMargin">
            <number>30</number>
           </property>
           <property name="rightMargin">
            <number>30</number>
           </property>
           <property name="bottomMargin">
            <number>30</number>
           </property>
           <property name="spacing">
            <number>11</number>
           </property>
           <item row="0" column="1">
            <widget class="QPushButton" name="btnSimGram">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>华文中宋</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>文法简化</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="btnRmFactor">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>华文中宋</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>消除左公因子</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="btnRmRecursion">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>华文中宋</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>消除左递归</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btnFirstFollow">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>华文中宋</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>First Follow</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="btnLL1">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>LL1分析表</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="btnTreeAnal">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>语法树分析</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>964</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
